Library/Homebrew/unpack_strategy/uncompressed.rb

```diff
@@ -22,7 +22,7 @@ def extract_nestedly(to: nil, basename: nil, verbose: false, prioritize_extensio
 
     sig { override.params(unpack_dir: Pathname, basename: Pathname, verbose: T::Boolean).returns(T.untyped) }
     def extract_to_dir(unpack_dir, basename:, verbose: false)
-      FileUtils.cp path, unpack_dir/basename, preserve: true, verbose:
+      FileUtils.cp path, unpack_dir/basename.sub(/^[\da-f]{64}--/, ""), preserve: true, verbose:
```

Comment:
I noticed during the audit signing that this caused audit to fail as the paths didn't exist.  This applied to direct binary downloads only.  Making this adjustment now causes the paths to be found.